packages/permission_handler/tizen/src/app_settings_manager.cc

```diff
@@ -12,6 +12,12 @@ static std::string ErrorToString(int error) {
       return "AppControl - Invalid parameter";
     case APP_CONTROL_ERROR_OUT_OF_MEMORY:
       return "AppControl - Out of memory";
+    case APP_CONTROL_ERROR_APP_NOT_FOUND:
```

Comment:
I think we can use ```get_error_message``` instead of ```ErrorToString```

Reply:
Brilliant. I didn't even know such function exists. We may make a separate PR to apply the same thing to other plugins (package_info, image_picker, share, ...) as well.